.github/workflows/test-push.yml

```diff
@@ -20,29 +20,58 @@ concurrency:
 jobs:
   build:
     runs-on: ubuntu-24.04
+    env:
+      GRADLE_OPTS: -Dorg.gradle.daemon=false -Xmx4g -XX:+UseParallelGC
```

Comment:
Why would you want to use the parallel GC!? no args best args imo (other than memory)